<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferencesDialog</class>
 <widget class="QDialog" name="PreferencesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>957</width>
    <height>677</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>585</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Tulip settings</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/tulip/gui/icons/logo32x32.png</normaloff>:/tulip/gui/icons/logo32x32.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton, QComboBox {
border-image: url(:/tulip/gui/ui/btn_26.png) 4;
border-width: 4;
padding: 0px 6px;
font-size: 12px;
}
QPushButton::flat {
border-image: none;
border-width: 4;
padding: 0px 6px;
font-size: 12px;
background-color: transparent;
}
QPushButton, QComboBox, QLabel {
color: black;
}
QPushButton:hover, QComboBox:hover {
border-image: url(:/tulip/gui/ui/btn_26_hover.png) 4;
}
QPushButton:disabled, QComboBox::disabled {
color:gray;
}
QPushButton:pressed{
border-image: url(:/tulip/gui/ui/btn_26_pressed.png) 4;
}
QPushButton::menu-indicator{
subcontrol-origin: margin;
subcontrol-position: center right;
right: 4px;
}
QPushButton {
outline: none;
margin: 2
}
QComboBox::down-arrow {
image: url(:/tulip/gui/ui/combobox_arrow.png);
}
QComboBox:drop-down {
subcontrol-origin: padding;
subcontrol-position: top right;
border-left-style: none; 
border-top-right-radius: 1px;
border-bottom-right-radius: 1px;
}
#PreferencesDialog {
background-color: white;
border: 0px;
}
QFrame[ section=&quot;true&quot; ] {
border-top: 1px solid #D9D9D9;
padding-top: 20px;
}
QFrame[ subsection = &quot;true&quot; ] {
border-top: 1px solid #D9D9D9;
padding-top: 5px;
background-color: #F4F4F4;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>10</number>
   </property>
   <item>
    <widget class="Line" name="line">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>1</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="horizontalFrame_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="title">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>36</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font-weight: bold;
color: #595959;</string>
        </property>
        <property name="text">
         <string>&lt;font size=&quot;5&quot;&gt;Settings&lt;/font&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="networkFrame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="section" stdset="0">
      <bool>true</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="networkFrameTitle">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font-weight: bold;</string>
        </property>
        <property name="text">
         <string>Proxy settings</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="networkFrameContents">
        <property name="spacing">
         <number>5</number>
        </property>
        <item>
         <widget class="QCheckBox" name="proxyCheck">
          <property name="styleSheet">
           <string notr="true">font: italic;</string>
          </property>
          <property name="text">
           <string>Enable application proxy</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="networkFrame1">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QComboBox" name="proxyType">
             <item>
              <property name="text">
               <string>SOCKS 5</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>HTTP</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Caching-only HTTP</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Caching FTP</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="networkFrame2">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Host:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="proxyAddr">
             <property name="placeholderText">
              <string>address</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="proxyPort">
             <property name="maximum">
              <number>65535</number>
             </property>
             <property name="value">
              <number>3128</number>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="networkFrame3">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QCheckBox" name="proxyAuthCheck">
             <property name="text">
              <string>Use authentification</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="proxyUser">
             <property name="enabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="proxyPassword">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="renderingFrame">
     <property name="section" stdset="0">
      <bool>true</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="renderingFrameTitle">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>85</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font-weight: bold;</string>
        </property>
        <property name="text">
         <string>Drawing defaults</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="renderingFrameContents">
        <property name="spacing">
         <number>10</number>
        </property>
        <item>
         <widget class="QTableWidget" name="graphDefaultsTable">
          <property name="styleSheet">
           <string notr="true">QAbstractItemView {
background-color: white;
border: 0px;
}
QHeaderView::section {
background-color: white;
height: 30px;
font: bold 10px;
color: black;
border: 0px;
}</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::AllEditTriggers</set>
          </property>
          <property name="showGrid">
           <bool>false</bool>
          </property>
          <property name="cornerButtonEnabled">
           <bool>false</bool>
          </property>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>200</number>
          </attribute>
          <attribute name="horizontalHeaderMinimumSectionSize">
           <number>1</number>
          </attribute>
          <row>
           <property name="text">
            <string>New Row</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>New Row</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>New Row</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>New Row</string>
           </property>
          </row>
          <column>
           <property name="text">
            <string/>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Nodes</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Edges</string>
           </property>
          </column>
          <item row="0" column="0">
           <property name="text">
            <string>Default color</string>
           </property>
           <property name="font">
            <font>
             <pointsize>8</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
          </item>
          <item row="1" column="0">
           <property name="text">
            <string>Default size</string>
           </property>
           <property name="font">
            <font>
             <pointsize>8</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
          </item>
          <item row="2" column="0">
           <property name="text">
            <string>Default shape</string>
           </property>
           <property name="font">
            <font>
             <pointsize>8</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
          </item>
          <item row="3" column="0">
           <property name="text">
            <string>Selection color (nodes = edges)</string>
           </property>
           <property name="font">
            <font>
             <pointsize>8</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="automaticFrame">
     <property name="section" stdset="0">
      <bool>true</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="automaticTitle">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font-weight: bold;</string>
        </property>
        <property name="text">
         <string>Automatic actions</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="automaticContents">
        <item>
         <widget class="QCheckBox" name="displayDefaultViews">
          <property name="text">
           <string>Display Node Link diagram and Spreadsheet views at create/import/load time</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="aspectRatioCheck">
          <property name="text">
           <string>Adjust aspect ratio when changing layout</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="viewOrthoCheck">
          <property name="text">
           <string>Use Orthogonal projection by default (Central projection if not)</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="colorMappingCheck">
          <property name="text">
           <string>Map colors when computing a metric</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="resultPropertyStoredCheck">
          <property name="toolTip">
           <string>The contents of the result property of an algorithm (viewMetric for example) will be stored in a new property automatically named using the values of the parameters of the algorithm</string>
          </property>
          <property name="text">
           <string>Store computed result property of an algorithm in a specific property</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="runningTimeComputedCheck">
          <property name="toolTip">
           <string>The time spent in an algorithm computation will be displayed in the message box</string>
          </property>
          <property name="text">
           <string>Compute time spent when running an algorithm</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>proxyCheck</sender>
   <signal>clicked(bool)</signal>
   <receiver>networkFrame1</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>538</x>
     <y>107</y>
    </hint>
    <hint type="destinationlabel">
     <x>538</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>proxyCheck</sender>
   <signal>clicked(bool)</signal>
   <receiver>networkFrame3</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>538</x>
     <y>107</y>
    </hint>
    <hint type="destinationlabel">
     <x>538</x>
     <y>205</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>proxyAuthCheck</sender>
   <signal>clicked(bool)</signal>
   <receiver>proxyUser</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>212</x>
     <y>190</y>
    </hint>
    <hint type="destinationlabel">
     <x>397</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>proxyAuthCheck</sender>
   <signal>clicked(bool)</signal>
   <receiver>proxyPassword</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>212</x>
     <y>190</y>
    </hint>
    <hint type="destinationlabel">
     <x>614</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>478</x>
     <y>446</y>
    </hint>
    <hint type="destinationlabel">
     <x>478</x>
     <y>235</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>478</x>
     <y>446</y>
    </hint>
    <hint type="destinationlabel">
     <x>478</x>
     <y>235</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>proxyCheck</sender>
   <signal>clicked(bool)</signal>
   <receiver>networkFrame2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>538</x>
     <y>107</y>
    </hint>
    <hint type="destinationlabel">
     <x>538</x>
     <y>173</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>settingsChanged()</slot>
  <slot>applySettings()</slot>
 </slots>
</ui>
